pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,20 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+In HTML, information that is programmatically determinable from a link includes [text nodes][] that are [included in the accessibility tree][] and are
+
+- sibling [phrasing content][] nodes in the same [paragraph][] as the link;
```

Comment:
Phrasing content doesn't take text alternatives into consideration. Nor does it consider pseudo element content. I'm not sure what the best way to go here is. Perhaps you can look at the accessible name of these elements, or if they don't have one the name from contents value ([Step 2F.i in accName 1.1](https://www.w3.org/TR/accname-1.1/#step2F.i)).

Reply:
The definition requires that nodes contributing to the programmatically determined context are text nodes included in the accessibility tree. I'll add to those requirements that they have a non-empty accessible name. With this, it is safe to remove the phrasing content requirement and leave it as "sibling nodes in the same paragraph as the link"